sql/core/src/test/resources/sql-tests/inputs/pgSQL/date.sql

```diff
@@ -0,0 +1,358 @@
+--
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+--
+-- DATE
+-- https://github.com/postgres/postgres/blob/REL_12_BETA1/src/test/regress/sql/date.sql
```

Comment:
PostgresSQL 12 Beta2 was released on 20th June.
- https://www.postgresql.org/about/news/1949/

Could you check `https://github.com/postgres/postgres/blob/REL_12_BETA1/src/test/regress/sql/date.sql` and update this PR (if needed)?

Reply:
Only these files changed:
![image](https://user-images.githubusercontent.com/5399861/60638151-d1255300-9e4f-11e9-9c3f-5101d5b32e27.png)

Reply:
Then, let's update this line 6 from `REL_12_BETA1` to `REL_12_BETA2` because we had better point to the up-to-date branch.

Reply:
btw, we don't need to have the same snapshot of these migrated pg tests? In `sql-tests/inputs/pgSQL`, already-merged tests comes from `REL_12_BETA1` and the other tests will come from `REL_12_BETA2`?

Reply:
Yes. I think so, @maropu . If we started with `REL_11`, it might be a different story. However, we don't need to stick to `BETA1`. It's unofficially one. We just picked up them because that was the latest one at that time.

Also, we can update them later consistently and easily. For now, it seems that the best effort is to keep using the latest one at the merge time.

Reply:
ok, thanks!